zkit/Cargo.toml

```diff
@@ -21,4 +21,9 @@ rand = "0.4"
 
 # error and log
 log = "0.4.0"
-env_logger = "0.10"
\ No newline at end of file
+env_logger = "0.10"
+
+
+[features]
+default = ["profiler"]
```

Comment:
Can we remove profiler from default feature?

Reply:
Sure